plotly/tests/test_optional/test_opt_tracefactory.py

```diff
@@ -0,0 +1,140 @@
5036, 1.055777305072455, None, 1.0630091915133837, 1.0717356090899524, 1.0692795518975082, None, 1.0751571208598627, 1.0841470984807897, 1.0800198416126761, None, 1.0843087729696688, 1.0932039085967227, 1.0875699927599589, None, 1.0900993003528923, 1.0985449729988461, 1.0916290046389945, None, 1.0922978529558298, 1.0999573603041506, 1.0920350572551183, None, 1.0908167814241361, 1.0973847630878195, 1.0887719625718983, None, 1.1946000000000001, 1.2000000000000002, 1.2054000000000002, None, 1.216689791150914, 1.2238403196954075, 1.2272745101915994, None, 1.2381142129963223, 1.2467302010770382, 1.2480616717315536, None, 1.2580191414459216, 1.2677570968074043, 1.2669327660869463, None, 1.2756110298160606, 1.286082730907943, 1.2831354622770101, None, 1.2901885450318353, 1.3009765181769477, 1.2960238099352113, None, 1.3011705275636027, 1.3118446903160674, 1.305083991311951, None, 1.3081191604234708, 1.3182539675986154, 1.3099548055667933, None, 1.3107574235469959, 1.3199488323649808, 1.310442068706142, None, 1.3089801377089636, 1.3168617157053837, 1.3065263550862782, None, 1.3937000000000002, 1.4000000000000001, 1.4063000000000001, None, 1.4194714230093997, 1.4278137063113088, 1.4318202618901992, None, 1.4444665818290428, 1.4545185679232113, 1.4560719503534791, None, 1.4676889983535752, 1.4790499462753051, 1.4780882271014373, None, 1.4882128681187374, 1.5004298527259334, 1.4969913726565116, None, 1.505219969203808, 1.5178059378731057, 1.5120277782577465, None, 1.5180322821575363, 1.5304854720354117, 1.5225979898639423, None, 1.5261390204940493, 1.5379629621983846, 1.5282806064945924, None, 1.5292169941381617, 1.5399403044258109, 1.5288490801571655, None, 1.5271434939937907, 1.5363386683229474, 1.5242807476006577, None, 1.5928, 1.6000000000000001, 1.6072000000000002, None, 1.6222530548678851, 1.6317870929272098, 1.636366013588799, None, 1.650818950661763, 1.6623069347693842, 1.6640822289754045, None, 1.6773588552612289, 1.6903427957432058, 1.6892436881159283, None, 1.7008147064214141, 1.7147769745439239, 1.7108472830360133, None, 1.7202513933757804, 1.7346353575692635, 1.7280317465802815, None, 1.73489403675147, 1.7491262537547563, 1.7401119884159342, None, 1.7441588805646278, 1.7576719567981538, 1.7466064074223913, None, 1.7476765647293278, 1.7599317764866409, 1.7472560916081892, None, 1.745306850278618, 1.7558156209405114, 1.7420351401150376, None, 1.7919, 1.8, 1.8081, None, 1.8250346867263707, 1.8357604795431111, 1.8409117652873987, None, 1.8571713194944832, 1.870095301615557, 1.87209250759733, None, 1.8870287121688825, 1.9016356452111065, 1.9003991491304193, None, 1.9134165447240905, 1.9291240963619141, 1.9247031934155145, None, 1.935282817547753, 1.9514647772654214, 1.9440357149028167, None, 1.9517557913454038, 1.9677670354741008, 1.957625986967926, None, 1.962178740635206, 1.9773809513979228, 1.9649322083501899, None, 1.9661361353204936, 1.9799232485474709, 1.9656631030592127, None, 1.9634702065634453, 1.9752925735580753, 1.9597895326294172, None],
```

Comment:
@theengineear yeah - but actually these optional tests are failing on circle (passing for me if I run nosetests and the core tests are passing on circle)